utils/cached_file.hh

```diff
@@ -326,6 +326,7 @@ public:
     }
 
     size_t evict_range(cache_type::iterator start, cache_type::iterator end) noexcept {
+      return with_allocator(standard_allocator(), [&] {
```

Comment:
Shouldn't the cached_page also be lsa-allocated? It can use many objects and create memory fragmentation. The btree is lsa aware.

If so, we can do this as a follow up.